ts-starter-complete-cotta/scripts/main.ts

```diff
@@ -41,7 +49,10 @@ function initializeBreakTheTerracotta() {
   let players = world.getAllPlayers();
 
   for (let player of players) {
-    player.runCommand("scoreboard players set @s score 0");
+    if (player.scoreboardIdentity === undefined) {
```

Comment:
nit: you can simply do `if (!player.scoreboardIdentity) { `

Reply:
This was already simplified. Just merge with latest main.